graph/common/Storage.java

```diff
@@ -1,219 +0,0 @@
-/*
- * Copyright (C) 2022 Vaticle
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
- *
- */
-
-package com.vaticle.typedb.core.graph.common;
-
-import com.vaticle.typedb.core.common.collection.ByteArray;
-import com.vaticle.typedb.core.common.collection.KeyValue;
-import com.vaticle.typedb.core.common.exception.ErrorMessage;
-import com.vaticle.typedb.core.common.exception.TypeDBException;
-import com.vaticle.typedb.core.common.iterator.sorted.SortedIterator.Forwardable;
-import com.vaticle.typedb.core.common.parameters.Order;
-import com.vaticle.typedb.core.graph.iid.InfixIID;
-import com.vaticle.typedb.core.graph.iid.VertexIID;
-
-import javax.annotation.Nullable;
-import java.util.Objects;
-import java.util.Optional;
-
-import static com.vaticle.typedb.common.util.Objects.className;
-import static com.vaticle.typedb.core.common.exception.ErrorMessage.Internal.ILLEGAL_CAST;
-import static com.vaticle.typedb.core.common.exception.ErrorMessage.Internal.UNRECOGNISED_VALUE;
-
-public interface Storage {
```

Comment:
moved into `//encoding` and split into `//encoding/Storage` and `//encoding/Key`